benchmarks/rayleigh_taylor_instability_free_surface/kaus_rayleigh_taylor_instability.prm

```diff
@@ -0,0 +1,162 @@
+# Input file for the 2D Rayleigh Taylor benchmark described
+# in Kaus (2010) and Rose et al. (2017).
+# The benchmark considers a denser, more viscous layer
+# overlying lighter, less viscous mantle. The interface
+# between the layers is perturbed, leading to drips along
+# the vertical domain boundaries. A free surface allows
+# for the development of surface topography. 
+ 
+############### Global parameters
+set Additional shared libraries            = librayleigh_taylor_instability.so
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 6e6
+# For a timestep of 2500 yr and no stabilitation,
+# the model is stable. For a timestep of 5000 yr,
+# it is not ("drunken sailor").
+set Maximum time step                      = 50e2 
+set Use years in output instead of seconds = true
+set CFL number                             = 0.5
+set Nonlinear solver tolerance             = 1e-8
```

Comment:
The nonlinear solver tolerance is meaningless since you use single Advection, single Stokes. Remove line.